lib/confirm_order.rb

```diff
@@ -0,0 +1,20 @@
+require 'twilio-ruby'
+
+class ConfirmOrder
+  
+  def confirm_order
+    # Your Account SID from www.twilio.com/console
+    account_sid = "AC0669ddc0d4125dde1e1f39d69ae96c44"
```

Comment:
This should be a constant, defined outside the method, either as a class variable or as a public variable in a separate 'config' class.